src/backend/utils/standardize.js

```diff
@@ -0,0 +1,25 @@
+const crypto = require('crypto');
```

Comment:
I don't think that we should let the key naming code leak out of `storage.js`.  It's a detail only for communication from Redis, and the rest of our code should be isolated from it.

If someone wants to get/set a feed, they think in terms of a `url`.  If someone wants to get/set a post, they think in terms of a `guid`.  We should do all the key namespacing quietly at the point of interaction with Redis, and then strip it out when we communicate things back to the user.

Reply:
The actual logic of what you're doing in here is great, I just want to hide it all from the rest of our code vs. making it a module you can require elsewhere.

Reply:
I moved all the logic to `storage.js`, and removed `standardize.js`.